File: dotnet-back-end/ktdi-app/Model/Feedback.cs
using Microsoft.AspNetCore.Authorization;
using Microsoft.AspNetCore.Mvc;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Authorize]
[ApiController]
[Route("api/[controller]")]
public class Feedback
{
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int FeedbackId { get; set; }

    public int UserId { get; set; }

    public int CollegeRating { get; set; }

    public int AccommodationRating { get; set; }

    public int FacilitiesRating { get; set; }

    public string Recommendation { get; set; }

    [ForeignKey("UserId")]
    public Authentication User { get; set; }
}

File: springboot-back-end/src/main/java/com/model/Feedback.java
package com.model;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Table(name = "Feedback")
@Data
public class Feedback {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "feedback_id")
    private Long feedbackId;

    @ManyToOne
    @JoinColumn(name = "user_id", nullable = false)
    private Authentication user;

    @Column(name = "college_rating")
    private Integer collegeRating;

    @Column(name = "accommodation_rating")
    private Integer accommodationRating;

    @Column(name = "facilities_rating")
    private Integer facilitiesRating;

    private String recommendation;
}